device/config/configurator.go

```diff
@@ -110,28 +102,296 @@ func (c *configurator) Write() error {
 }
 
 // AddConfigFn
-func (c *configurator) AddConfigFn(ccf *CallbackFn) {
-	c.order = append(c.order, ccf)
+func (c *Configurator) AddConfigFn(name string, ccf *CallbackFn) {
+	c.order[name] = ccf
 }
 
-// SetConfigFn sets CallbackFn of a specified number of the array
-func (c *configurator) SetConfigFn(num int, ccf *CallbackFn) {
-	c.order[num] = ccf
+// SetConfigFn sets CallbackFn of the specified name
+func (c *Configurator) SetConfigFn(name string, ccf *CallbackFn) {
+	c.order[name] = ccf
 }
 
-// GetConfigFn returns GetConfigFn from the array by a number
-func (c *configurator) GetConfigFn(num int) *CallbackFn {
-	return c.order[num]
+// GetConfigFn returns configuration function by it's name
+func (c *Configurator) GetConfigFn(name string) *CallbackFn {
+	return c.order[name]
 }
 
 // RemoveConfigFn removes CallbackFn from order
-func (c *configurator) RemoveConfigFn(num int) {
-	if num == len(c.order)-1 {
-		c.order = c.order[:num]
+func (c *Configurator) RemoveConfigFn(name string) {
+	delete(c.order, name)
+}
+
+func (c *Configurator) StoreValue(name string, value interface{}) {
```

Comment:
__[golint]__
* [golint] exported method Configurator.StoreValue should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->